packages/react-components/react-theme/src/global/colors.ts

```diff
@@ -848,54 +848,95 @@ const charcoal: ColorVariants = {
   tint60: '#f7f7f7',
 };
 
-export const sharedColors: GlobalSharedColors = {
-  darkRed,
-  burgundy,
-  cranberry,
+export type StatusSharedColors = 'red' | 'green' | 'darkOrange' | 'yellow' | 'berry';
+
+export const statusSharedColors: Pick<GlobalSharedColors, StatusSharedColors> = {
   red,
+  green,
   darkOrange,
-  bronze,
+  yellow,
+  berry,
+};
+
+export type PersonaSharedColors =
```

Comment:
As a follow up task, the internal structure of the file must be refactored and token pipeline updated.